addons/website_profile/views/website_profile.xml

```diff
@@ -129,7 +129,7 @@
                             <div class="row">
                                 <input name="user_id" t-att-value="user.id" type="hidden"/>
                                 <div class="form-group col-12">
-                                    <label class="text-primary mb-1 d-block" for="user_name"><span class="font-weight-bold">Real name</span></label>
+                                    <label class="text-primary mb-1 d-block" for="user_name"><span class="font-weight-bold">Name</span></label>
```

Comment:
@mart-e translations will be broken and he will incinerate you with burning cheese. Can be renamed in master if necessary.

Reply:
@github we need a :cheese: reaction to messages !